lib/Solver/BitwuzlaSolver.cpp

```diff
@@ -771,7 +771,7 @@ class BitwuzlaIncNativeSolver {
   Bitwuzla &getOrInit();
 
   bool isConsistent() const {
-    klee_warning("Empty isConsistent() check");
+    // klee_warning("Empty isConsistent() check");
```

Comment:
I think we should attach a comment to this function that will describe what kind of check is to be expected here in the future (or remove this function altogether).